### DISCUSSION TITLE
How to disable Zip feature?

### DISCUSSION THREAD

**Discussion by rocheston** [DustinBrett/daedalOS#308] [Feature Requests]
I was unable to locate the required information on your website about creating shortcuts for applications, particularly Chrome and Terminal, on the desktop. Could you please provide guidance on how to perform this action? Your help would be greatly appreciated.

I tried this, doesn't work.

[InternetShortcut]
IconFile=/System/Icons/chromium.webp
URL=https://example.com/?app=Browser

One more question. How to disable Zip feature. I don't want the users to zip files and download them. There are tons of files that are huge like 1.5GB.

**Comment by rocheston** [comment]
[InternetShortcut]
BaseURL=Browser
Comment=Access the Internet
IconFile=/System/Icons/chromium.webp

Figured it out. How about zip question?

**Comment by DustinBrett** [comment]
It depends what you mean by zip feature. If you mean the "Add to archive" or "Extract here" options, these can be removed in useFileContextMenu. If you mean the ability to open zip files, that is more complex but can be done by removing any code that mounts ZipFS.

**Comment by DustinBrett** [comment]
> [InternetShortcut] BaseURL=Browser Comment=Access the Internet IconFile=/System/Icons/chromium.webp
> 
> Figured it out. How about zip question?

Also for this shortcut, you can probably skip IconFile and it will use the icon associated with the BaseURL app.

**Comment by DustinBrett** [comment]
I will move this to feature requests as it's not an Issue so much and there is no way to disable Zip as is. We can discuss how to do it manually when you have time.
